untraced/yksg/src/lib.rs

```diff
@@ -18,7 +18,7 @@ use ykpack::{
 use yktrace::sir::{RETURN_LOCAL, SIR};
 
 /// Stores information needed to recreate stack frames in the StopgapInterpreter.
-pub struct FrameInfo {
+pub struct IncomingFrame {
```

Comment:
Not sure about the name `IncomingFrame`, but honestly don't have a better idea either. @vext01 is good at naming stuff.

Reply:
When the stop-gap interpreter pushes a new frame, will it also be of this same type?

In that context, the frame isn't "incoming", and in fact I find the new name just raises questions.

Was there  a reason this can't just be `Frame`?

Reply:
I see, we already have a `Frame` below.

If this struct is only used during initialisation, then `IncomingFrame` makes sense, but how about `InitFrame`?

Reply:
It's "incoming" in the sense that it's passed to us from a compiled trace. We then convert it to a different (internal) format.

Reply:
Yeah, I eventually got there. I'm OK with `IncomingFrame` but prefer `InitFrame`. You guys fight it out :)

Reply:
I'm not a fan of `InitFrame`: it sounds like it's part of this module. I'm not promising that `IncomingFrame` is the best name ever, however!

Reply:
I'm fine with `IncomingFrame`, at least it's very verbose about what it does. I was hoping to find something snappier, but if we can't it's not the end of the world.

Reply:
I see.

In that case, the only other suggestion I have is "FailureFrame", but I'm not sure it's any better than your suggestion.